package.json

```diff
@@ -514,6 +514,7 @@
   "dependencies": {
     "ansi-colors": "3.2.3",
     "browser-stdout": "1.3.1",
+    "chokidar": "2.1.6",
```

Comment:
In v7.0.0 we drop Node v6, so `chokidar` v3 is ok.

Reply:
Updated in https://github.com/mochajs/mocha/pull/3980/commits/f18a0b66b5df455fc7b1598797e0e3245df52826